backend/node_modules/.package-lock.json

```diff
@@ -2389,7 +2389,17 @@
     "node_modules/asap": {
       "version": "2.0.6",
       "resolved": "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz",
-      "integrity": "sha512-BSHWgDSAiKs50o2Re8ppvp3seVHXSRM44cdSsT9FfNEUUZLOGWVCsiWaRPWM1Znn+mqZ1OfVZ3z3DWEzSp7hRA=="
```

Comment:
Was this folder `node_module` committed by mistake? 

it looks like we have committed the node_modules, we need to delete it and push. Then after add it to the the .ignore file.

Reply:
i think this is the one file from node_modules that has to be committed to source control according to https://docs.npmjs.com/cli/v8/configuring-npm/package-lock-json, 

im guessing the `.` before the package-lock means it ignores the git ignore and just commits that file, so yeah i think it is needed

Reply:
We need to keep the `backend/package-lock.json`, not this one which is inside the `node_modules` folder.
We now have our entire node_module folder committed to github: you can see it here: https://github.com/SyntheticDBX/Help-Others/tree/HO-15-Backend-organisation-schema/backend/node_modules this means we have thousands for files in giithub.

Reply:
Just seen that the node_module folder is on main already so we can discuss removing it